lib/manageiq/api/client/action.rb

```diff
@@ -0,0 +1,17 @@
+module ManageIQ
+  module API
+    module Client
+      class Action
+        attr_accessor :name
+        attr_accessor :method
+        attr_accessor :href
+
+        def initialize(action_hash)
```

Comment:
Is there a way we can avoid hashes as parameters?  I'm wondering if we can use Ruby keyword args here.  Then the caller can still call with a Hash, but we get the benefit of named args + enforced non-optional values.

```ruby
def initialize(name:, method:, href:, **rest)
  @name   = name
  @method = method
  @href   = href
end
```